lib/core/security/userRepository.js

```diff
@@ -290,8 +290,20 @@ class UserRepository extends Repository {
       throw kerror.get('security', 'user', 'anonymous_profile_required');
     }
 
-    await this.persistToDatabase(user, databaseOptions);
-    await this.persistToCache(user, cacheOptions);
+    try {
+      await this.persistToDatabase(user, databaseOptions);
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.log('DB');
```

Comment:
Are these intentionals?

Reply:
No it's just for debugging, btw I should get back to this bug!